src/components/Button/Button.tsx

```diff
@@ -43,6 +43,8 @@ export interface ButtonProps extends BaseButton {
   monochrome?: boolean;
   /** Icon to display to the left of the button content */
   icon?: React.ReactElement | IconSource;
+  /** Position of the Icon, defaults to left */
+  iconPosition?: 'left' | 'right';
```

Comment:
Fun question: how does the ordering of the icon work with right-to-left languages? Is there some language we can use here that doesn't assume left-to-right languages?

Reply:
Good point!

Reply:
@BPScott What do you think about "before" and "after" instead? I also added both cases to the readme. Can you approve the percy build?